### ISSUE TITLE
Build local bias track from control with paired-end mode

### ISSUE THREAD

**Issue by xiaoxuanzhu** [macs3-project/MACS#560]
I'm using MACS2 (v2.2.7.1) to analyze ChIP-seq data. Both of the ChIP sample and the input control are sequenced with **paired-end** mode.

I run `macs2 callpeak` with `-f BAMPE` and the program worked well. MACS2 was run in paired-end mode and the fragment size was detected based on the read pairs. But from the process information, I realized that the mean fragment size of control was ignored. Below is a part of process information I got:

```
#1 mean fragment size is determined as 282.9 bp from treatment 
#1 note: mean fragment size in control is 207.6 bp -- value ignored 
#1 fragment size = 282.9 
#1  total fragments in treatment: 114306603 
#1 user defined the maximum fragments... 
#1 filter out redundant fragments by allowing at most 1 identical fragment(s) 
#1  fragments after filtering in treatment: 114306603 
#1  Redundant rate of treatment: 0.00 
```
While I'm trying to figure out how exactly the program deal with the control sample, I found the Wiki page about MACS2 subcommands. [https://github.com/macs3-project/MACS/wiki/Advanced%3A-Call-peaks-using-MACS2-subcommands](Advanced: Call peaks using MACS2 subcommands) This is very helpful, but unfortunately it does not work for paired-end data.

My question is, is lambda track for control sample still calculated in single-end mode even if `-f BAMPE` is set?

**Comment by kalavattam** [comment]
Yes—see [here](https://github.com/macs3-project/MACS/issues/356#issuecomment-653692098).
